kernel/data-pipeline/cdc/client/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/client/handler/CDCRequestHandler.java

```diff
@@ -92,21 +90,18 @@ private void sendStartStreamingDataRequest(final ChannelHandlerContext ctx, fina
     
     private void processDataRecords(final ChannelHandlerContext ctx, final DataRecordResult result) {
         List<Record> recordsList = result.getRecordsList();
-        for (Record each : recordsList) {
```

Comment:
`getRecordsList` could be `getRecordList` (`repeated Record record` in protobuf)